certified-connectors/WenDocs Linker/apiDefinition.swagger.json

```diff
@@ -0,0 +1,363 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "WenDocs Linker",
+    "description": "Connect to WenDocs Publish API Service. Empowers registered users to call service to assemble WenDocs template(made by our Word add-in: Wendocs Template Designer) and json data, then publish to Word, PDF, HTML formats. By integrating with other connecters, it is easy to produce documents driven by data automatically, like a welcome letter, resume, business report...",
+    "contact": {
+      "name": "WenDocs Support",
+      "url": "https://www.wendocs.com/#/support",
+      "email": "support@wendocs.com"
+    }
+  },
+  "host": "api.wendocs.com",
+  "basePath": "/",
+  "schemes": ["https"],
+  "consumes": ["application/json"],
+  "produces": ["application/json"],
+  "x-ms-connector-metadata": [
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.wendocs.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.wendocs.com/public/PrivacyPolicy_en.html"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Business Management;Productivity"
+    }
+  ],
+  "paths": {
+    "/api/dynamicdoc/docx": {
+      "post": {
+        "summary": "Publish a Docx file",
+        "description": "Publish a docx file with template and json data",
+        "operationId": "PublishDocx",
+        "parameters": [
+          {
+            "name": "request body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "docName": {
+                  "type": "string",
+                  "description": "name of document"
+                },
+                "documentTemplateData": {
+                  "type": "string",
+                  "description": "a base64 string represents binary data of template file"
+                },
+                "jsonData": {
+                  "type": "string",
+                  "description": "customer data in json format"
+                },
+                "logLevel": {
+                  "type": "string",
+                  "description": "level of publish log: DEBUG, ERROR, WARNING, INFO"
+                },
+                "language": {
+                  "type": "string",
+                  "description": "supported language code of error message: en, zh"
+                },
+                "country": {
+                  "type": "string",
+                  "description": "supported country code of error message: US, CN"
+                },
+                "clientType": {
+                  "type": "string",
+                  "description": "For API service users, please use 'API Service'"
+                }
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "response body",

```

Comment:
@JulioWendocs Can you please provide valid descriptions for all the response codes.